### DISCUSSION TITLE
bgs, effects, colors, etc

### DISCUSSION THREAD

**Discussion by city41** [city41/smaghetti#148] [General]
![image](https://user-images.githubusercontent.com/141159/138776690-ede56c22-a0fb-433f-b6ed-2cc35a6579ab.png)

**Comment by city41** [comment]
![image](https://user-images.githubusercontent.com/141159/138776738-38be42e6-93bb-47a2-a6cd-c6f175bb895f.png)

**Comment by city41** [comment]
all were created/captured by @lelegofrog

**Comment by city41** [comment]
![image](https://user-images.githubusercontent.com/141159/138776840-1fe3e1a2-f4d4-4528-a5ad-d562b5623611.png)

bg 27

**Comment by city41** [comment]
![image](https://user-images.githubusercontent.com/141159/138776858-436062d6-544c-458e-975a-84d498bc39bd.png)

bg 28

**Comment by city41** [comment]
![image](https://user-images.githubusercontent.com/141159/138776869-f6ffe91f-6872-49c5-bb1e-9a697a788b68.png)

bg 2e

**Comment by city41** [comment]
![image](https://user-images.githubusercontent.com/141159/138776959-9c23b522-0973-4c0d-b91c-2b1735b4f683.png)


bg 30

**Comment by city41** [comment]
![image](https://user-images.githubusercontent.com/141159/138776987-6a85c9dd-2dd3-4f6c-9886-494a519ec238.png)

bg 2f 

obviously not legit but might as well record it

**Comment by city41** [comment]
![image](https://user-images.githubusercontent.com/141159/138777203-66cb4bcd-8083-4415-a4e5-6059fb336b9f.png)

bg 12

**Comment by lelegofrog** [comment]
![image](https://user-images.githubusercontent.com/61850335/139183426-44fd555e-2e7c-481f-b89c-e0b55ddd4e97.png)
![image](https://user-images.githubusercontent.com/61850335/139183440-62456011-2dfa-4d04-bc03-260bdf07cc8f.png)

**Reply by city41** [reply]
colorful brick wall is 0x31
